docs/monoDrive_home/Maps.md

```diff
@@ -78,7 +78,7 @@ These two maps display monoDrive using lane detection software using opendrive.
 
 Two Carla maps Town04 and Town07 are available in monoDrive Simulator or Scenario Editor. These maps have been updated with monoDrive physical materials, lighting, semantic labeling, dynamic weather, traffic signaling, and ray tracing settings.
 
-To see a tutorial on how to import a Carla Map, See [Carla to monoDrive Conversion](https://docs.google.com/document/d/18JTYXOFPb98qLm8aHvCYH4nb2eU6P5sVzD1qkONqTeU/edit).
+To see a tutorial on how to import a Carla Map, contact monoDrive Support at support@monodrive.io
```

Comment:
I'll suggest using lower case support _"monoDrive support". Same for other places to be consistent